src/api/archive_unstable_body.md

```diff
@@ -1,3 +1,11 @@
 # archive_unstable_body
 
-**TODO**
+**Parameters**:
+
+- `hash`: String containing the hexadecimal-encoded hash of the body to retreive.
```

Comment:
```suggestion
- `hash`: String containing the hexadecimal-encoded hash of the body to retrieve.
```
Nit: Its pretty clear what is meant here, but for first-time readers it may sound like they should  supply the hash of the body instead of the hash of the header here.